src/sentry/utils/pytest/selenium.py

```diff
@@ -312,6 +312,21 @@ def percy(request):
     return percy
 
 
+start_chrome_retry_attempts = 5
+
+
+def start_chrome(**chrome_args):
+    try:
+        return webdriver.Chrome(**chrome_args)
+    except WebDriverException as e:
+        global start_chrome_retry_attempts
+        if start_chrome_retry_attempts > 0:
```

Comment:
could use https://github.com/getsentry/sentry/blob/master/src/sentry/utils/retries.py#L28 if you wanted

Reply:
Good call